pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Repair damaged or corrupted `pcap` and `pcapng` files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a `pcap`/`pcapng` file:
+
+`pcapfix {{test.pcap}}`
+
+- Repair entire `pcap`/`pcapng` file:
```

Comment:
```suggestion
- Repair an entire `pcap`/`pcapng` file:
```

Reply:
Thanks for the suggestions! I should now have improved everything.